Modules/Loadable/Segmentations/EditorEffects/Python/SegmentEditorEffects/SegmentEditorThresholdEffect.py

```diff
@@ -72,11 +75,11 @@ def icon(self):
         return qt.QIcon()
 
     def helpText(self):
-        return """<html>Fill segment based on source volume intensity range<br>. Options:<p>
+        return _("""<html>Fill segment based on source volume intensity range<br>. Options:<p>
```

Comment:
```suggestion
       return _("""<html>Fill segment based on source volume intensity range.<br>Options:<p>
```

Reply:
For sake of consistency: `Options:` -> `Instructions:` ?

Reply:
`Instructions` is used when we describe a list of steps that the user should follow. `Option` refers to that you choose one of the listed options.